requirements-travis.txt

```diff
@@ -11,7 +11,7 @@ PyYAML==3.11
 Pillow==2.2.1
 snakefood==1.4
 networkx==1.9.1
-pygraphviz==1.3rc2
+#pygraphviz==1.3rc2
```

Comment:
Yep, no problem. Btw if you install `pygraphviz` manually, pip should recognize and skip installing it again. (I have similar problems, the graphviz check is always skipped as my version of `dot_parser` is incompatible. I simply ignore it as it's run in travis, the rest of the tests work well)